src/Sarif.Multitool.FunctionalTests/TestData/DeserializationError_Expected.sarif

```diff
@@ -33,7 +33,7 @@
                   "uri": "file:///C:/Code/sarif-sdk/bld/bin/Sarif.Multitool.FunctionalTests/AnyCPU_Debug/netcoreapp2.0/TestData/DeserializationError.sarif"
                 },
                 "region": {
-                  "startLine": 15,
+                  "startLine": 13,
```

Comment:
>13 [](start = 31, length = 2)

Larry, I can't account for this line change. Does it make sense to you?

Reply:
There are two things happening:
1. We've changed Newtonsoft versions since the last time we regenerated this file, and
2. The test is not sensitive to `startLine`.

The error JSON1002 is "Required property is missing.". The arguments (Line 23) tell us that the required `frames` property of the `stack` object is missing, which is true. The `startLine` is where Newtonsoft thinks the offending object lives. And in fact, (13, 13) (the position immediately before the open brace of the empty `frame` object) is a better answer than (15, 13) (a terrible answer -- past the end of the line containing the closing square brace of the `frames` array).

---
In reply to: [243702036](https://github.com/Microsoft/sarif-sdk/pull/1170#discussion_r243702036) [](ancestors = 243702036)